esti/migrate_test.go

```diff
@@ -169,13 +169,16 @@ func preMigrateTests(t *testing.T) {
 }
 
 func postMigrateTests(t *testing.T) {
+	defer deleteRepository(context.Background(), t, migrateStateRepoName)
```

Comment:
I have some concerns using the deleteRepository function directly.
What if I want to keep the repository for debugging purposes?

Reply:
if It's for debugging, you can comment out the deletions. The problem is that if I do not delete those repos, the next run will fail on an existing installation